common/vendors.js

```diff
@@ -25,3 +25,6 @@ require('qrcode.react');
 require('bignumber.js');
 require('classnames');
 require('./vendor/trezor-connect');
+require('./vendor/u2f-api');
+require('./vendor/ledger3');
+require('./vendor/ledger-eth');
```

Comment:
I know these are called vendor, but should we be adding these here? They're in version control, so even if they change, we won't generate a new DLL. I think they should remain in the main bundle.

Reply:
Not sure, I threw them in because I saw `trezor-connect` was included, though I think you're right. @HenryNguyen5, could you confirm?

Reply:
Yeah, `./vendor/*` should be removed for now, my bad.